s3/s3_transport/include/managed_shared_memory_object.hpp

```diff
@@ -6,6 +6,8 @@
 #include <boost/interprocess/managed_shared_memory.hpp>
 #include <boost/container/scoped_allocator.hpp>
 
+#include <rodsLog.h>
```

Comment:
I think this should be in quotation marks instead of brackets

Reply:
Why?  Quotes search for local directory first.  Brackets does not.  Seems like brackets is better?

Reply:
We discussed this.  It will be left as <rodsLog.h> for now.  Once Markus reorganizes the repositories this will be <irods/rodsLog.h>.